sway-lib-std/src/context/call_frames.sw

```diff
@@ -12,6 +14,10 @@ use ::contract_id::ContractId;
 
 const SAVED_REGISTERS_OFFSET: u64 = 64;
 const PREV_FRAME_POINTER_OFFSET: u64 = 48;
+/// Where 584 is the current offset in bytes from the start of the call frame.
+const FIRST_PARAMETER_OFFSET: u64 = 584;
+/// Where 592 is the current offset in bytes from the start of the call frame.
+const SECOND_PARAMETER_OFFSET: u64 = 592;
```

Comment:
I wonder if this should be `const SECOND_PARAMETER_OFFSET: u64 = FIRST_PARAMETER_OFFSET + 8;`

Reply:
Strange, I thought this would work given #2206